src/test/java/net/openhft/chronicle/queue/TestDeleteQueueFile.java

```diff
@@ -226,6 +228,55 @@ public void tailingThroughDeletedCyclesWillRefreshThenRetry_ReadOnly() throws IO
                 .build());
     }
 
+    @Test
+    public void deletingFilesChaosTest() throws InterruptedException {
```

Comment:
Does it actually do any assertions? What does it check for?

Reply:
We should be able to handle roll cycles going missing mostly gracefully, but interesting to understand the circumstances that we don't cover